bindings/nodejs/package.json

```diff
@@ -59,15 +59,12 @@
     "node": ">= 10"
   },
   "scripts": {
-    "artifacts": "napi artifacts",
```

Comment:
Why not keep this?

Reply:
We will never do publish by hand, so their no need for such script in `package.json`.